src/storage-control/src/generated/gapic_control/transport.rs

```diff
@@ -0,0 +1,512 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Code generated by sidekick. DO NOT EDIT.
+
+use crate::Result;
+#[allow(unused_imports)]
+use gax::error::Error;
+
+const DEFAULT_HOST: &str = "https://storage.googleapis.com";
+
+mod info {
+    const NAME: &str = env!("CARGO_PKG_NAME");
+    const VERSION: &str = env!("CARGO_PKG_VERSION");
+    lazy_static::lazy_static! {
+        pub(crate) static ref X_GOOG_API_CLIENT_HEADER: String = {
+            let ac = gaxi::api_header::XGoogApiClient{
+                name:          NAME,
+                version:       VERSION,
+                library_type:  gaxi::api_header::GAPIC,
```

Comment:
I just realized: if we are saying `has-veneer = true` this should be `gaxi::api_header:GCCL`. Please fix in a separate PR.

Reply:
hmm... ok. It is kind of a GAPIC disguised as a veneer.

Reply:
But yeah, we would want to do that for any other veneer.